docs/experimental/examples/endpoints_and_collections/create_guest_collection/index.rst

```diff
@@ -2,8 +2,60 @@
 Creating a Guest Collection
 ===========================
 
-TODO
+Within the Globus Ecosystem, data is managed through the abstraction of *Collections*.
+The example included on this page demonstrate how to create specifically a
+*Guest Collection* using the Globus Python SDK. Guest collections, formerly known as
```

Comment:
Is the desire to have italic, or bold words here?

Reply:
"Emphasized", so italics. That's how it rendered, no?

Reply:
It did!